src/lplib/option.ml

```diff
@@ -38,3 +37,16 @@ let eq : 'a eq -> 'a option eq = fun eq_elt o1 o2 ->
 
 let pp : 'a pp -> 'a option pp = fun elt ppf o ->
   match o with None -> () | Some e -> elt ppf e
+
+module Monad = struct
+  let ( let* ) = Stdlib.Option.bind
```

Comment:
Could you please add a comment explaining how to use this notation. "let* x = t in u" is the same as "bind ...".